src/main/java/emu/grasscutter/server/packet/send/PacketGetShopRsp.java

```diff
@@ -42,9 +42,7 @@ public PacketGetShopRsp(Player inv, int shopType) {
                         .setEndTime(info.getEndTime())
                         .setMinLevel(info.getMinLevel())
                         .setMaxLevel(info.getMaxLevel())
-                        .setMcoin(info.getMcoin())
-                        .setDisableType(info.getDisableType())
-                        .setSecondarySheetId(info.getSecondarySheetId());
```

Comment:
not sure its good to flatout remove them, they are used for multiple event shops (even newer once)

Reply:
will comment them out instead, these fields weren't available when i went to build because they were removed for 'deprecation' reasons by the team which does definitions

Reply:
yeah, I don't think its deprecated, when its still defined for example here in an 3.5 event:
https://gitlab.com/YuukiPS/GC-Resources/-/blob/3.6/Resources/ExcelBinOutput/ShopGoodsExcelConfigData.json#L27160

But its not used outside of activity shops, as far as I see, so its easy to overlook.

So yeah, commenting it out looks like the better solution ^^